### PR TITLE
XOR-51 [FIX] Ensure link label text not clear after clicking on textbox

### PR DESCRIPTION
**Pull request by amolsuryawanshi47**
**Product areas affected**
Text Component

**What does this PR do?**
link label text not clear after clicking on textbox 

**JIRA ticket**
[JIRA ticket](https://weboo.atlassian.net/browse/XOR-51)

**Result**

XOR-51

**Checklist**
 None

**Testing instructions**
None

**Deployment instructions**
None

**Author concerns**
None

### CODE REVIEW

**Review by NikitaJadhavXoriant — APPROVED** [review]
Code looks ok to me

File: js/build.js [hunk 1]
```diff
@@ -260,7 +260,7 @@
             });
 
             ed.on('focus', function() {
-              if (!hasValue) {
+              if (!widgetData.html) {
```
Comment (tonytlwu) [inline js/build.js:263]: `var hasValue = !!widgetData.html;` is declared at the top, which means `!hasValue` is the same as `!widgetData.html`. 

Why is this change required?

**Review by tonytlwu — CHANGES_REQUESTED** [review]

File: js/build.js (hunk 1 above)
Comment (amolsuryawanshi47) [reply js/build.js:263]: Yes, you are right. but it is getting set in on new instance creation, where as we need to check value when focus is enter on old instance (without refresh) of textbox.

when we use the `!widgetData.html` condition then we getting the expected behavior in both case.

Comment (tonytlwu) [reply js/build.js:263]: 👍🏻

**Review by tonytlwu — APPROVED** [review]
